### ISSUE TITLE
upgrading to version `3.4.2` failing because of postinstall script in package.json

### ISSUE THREAD

**Issue by alimek** [freshollie/jest-dynalite#69]
When i try to upgrade to 3.4.2 i am getting this error when executing `yarn upgrade`:

```
[4/4] ⡀ jest-dynalite
error /Users/XX/Sites/XX/XX/node_modules/jest-dynalite: Command failed.
Exit code: 1
Command: yarn build && husky install
Arguments:
Directory: /Users/X/Sites/XX/XX/node_modules/jest-dynalite
Output:
yarn run v1.22.10
$ rm -rf dist && tsc -p tsconfig.build.json
error TS5058: The specified path does not exist: 'tsconfig.build.json'.
```

**Comment by freshollie** [comment]
🤦  thanks for reporting. I'll get a quick fix in

**Comment by freshollie** [comment]
Can you try `3.4.3` 🙏

**Comment by alimek** [comment]
still getting same error

**Comment by freshollie** [comment]
Ok found a workaround in this thread will try again: https://github.com/typicode/husky/issues/941

Seems the husky docs are not correct for `yarn npm publish`

**Comment by freshollie** [comment]
Ok, 🤞  this is fixed in `v3.4.4`

**Comment by alimek** [comment]
yup, its working now. Thanks for quick fix 👍
